http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/CityListWrapperSerializer.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import java.io.ByteArrayInputStream;
+import java.io.StringWriter;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.xml.bind.JAXBContext;
+import jakarta.xml.bind.JAXBException;
+import jakarta.xml.bind.Marshaller;
+import jakarta.xml.bind.Unmarshaller;
+import jakarta.xml.bind.annotation.XmlRootElement;
+
+import org.slf4j.Logger;
```

Comment:
we use JBOSS logger, right?

Reply:
Oh, I see, in vert.x test suite we used to use slf4j, changing it to JBOSS...